.github/workflows/example_tests.yml

```
@@ -33,63 +33,43 @@ jobs:
-        example: [gpt-oss, llm_distill, llm_qat, llm_sparsity, specdec_bench]
-        include:
-          - example: speculative_decoding
-            docker_image: "26.01"
+        example: [llm_distill, llm_qat, llm_sparsity, specdec_bench, speculative_decoding]
     uses: ./.github/workflows/_example_tests_runner.yml
     permissions:
       contents: read
     secrets: inherit
     with:
-      docker_image: "nvcr.io/nvidia/pytorch:${{ matrix.docker_image || '26.06' }}-py3"
+      docker_image: "nvcr.io/nvidia/pytorch:${{ matrix.docker_image || '26.07' }}-py3"
```

actionlint (1.7.12)

[error] property "docker_image" is not defined in object type {example: string}

(expression)